examples/src/test/kotlin/BuildersTest.kt

```diff
@@ -75,8 +74,8 @@ internal class BuildersTest {
         // :snippet-start: builders
         data class Results(val email: String)
 
-        val filter = and(eq(User::gender.name, "female"), gt(User::age.name, 29))
-        val projection = fields(excludeId(), include("email"))
+        val filter = Filters.and(Filters.eq(User::gender.name, "female"), Filters.gt(User::age.name, 29))
+        val projection = Projections.fields(Projections.excludeId(), Projections.include("email"))
```

Comment:
Q: Do you know whether we have a standard around including Filters/Projects/etc. in these operations for readability? If not, does the compiler require these to be here?

Reply:
I had a conversation with Cory about this, she said we wanted to keep the class names when calling the methods. I'm not sure the reasoning, but that **is** the convention for this documentation